### PR TITLE
fix(client-debugger): Fix emitVisualUpdate to sync container and devTools 

### PR DESCRIPTION
**Pull request by jikim-msft**
#### Description

https://dev.azure.com/fluidframework/internal/_workitems/edit/4194/

This PR fixes the error of sending promise object to message passing which results in disconnect of container and disalignment of container state and `devTools` panel. 

#### Before
image

#### After
image

### CODE REVIEW

File: packages/tools/client-debugger/client-debugger/src/data-visualization/DataVisualization.ts [hunk 1]
```diff
@@ -344,8 +346,9 @@ export class VisualizerNode extends TypedEventEmitter<DataVisualizerEvents> impl
 	 * {@link SharedObjectListenerEvents | "update"} event.
 	 */
 	private emitVisualUpdate(): void {
-		const visualTree = this.render();
-		this.emit("update", visualTree);
+		this.render().then((visualTree: FluidObjectNode) => {
+			this.emit("update", visualTree);
+		}, console.error);
```
Comment (alexvy86) [inline packages/tools/client-debugger/client-debugger/src/data-visualization/DataVisualization.ts:351]: Found the Promise 😄 . However, did we explore the option of making `emitVisualUpdate` async and awaiting the call to `this.render()`? There's a few call sites for it, so maybe knowing that the actual emit of the "update" event is an async thing would help prevent other bugs.

E.g. event listeners that call this and then `return true;` to indicate that they handled the message (`onOpHandler` on line 292). It feels like that should await the call to `emitVisualUpdate` before returning.

**Comment by Josmithr** [comment]
Let's be sure to get the regression test added alongside these changes. They could go in as a separate PR if we're in a rush to unblock, but generally it's good practice to include the regression test alongside the fix.

File: packages/tools/client-debugger/client-debugger/src/data-visualization/DataVisualization.ts (hunk 1 above)
Comment (Josmithr) [reply packages/tools/client-debugger/client-debugger/src/data-visualization/DataVisualization.ts:351]: That's a fair point. It puts the burden on the callers to handle the promise, but that might be better (and there are only 2 callers).

Comment (jikim-msft) [reply packages/tools/client-debugger/client-debugger/src/data-visualization/DataVisualization.ts]: Added async / await for `onOpHandler`.

**Review by alexvy86 — APPROVED** [review]

**Pull request merged**